### ISSUE TITLE
Solving as a "Full House" instead of "Four of a Kind".

### ISSUE THREAD

**Issue by acidicchip** [goldfire/pokersolver#11]
The following will give me "Four of a Kind, A's" as expected:
`Hand.solve(['Ad', 'As', 'Jc', 'Th', '2d', '0r', '0r']).descr;`

However, the following is giving me "Full House, 10's over A's" instead of the expected "Four of a Kind, A's":
`Hand.solve(['Ad', 'As', 'Ac', 'Th', '2d', '0r', '0r']).descr`

**Comment by goldfire** [comment]
I assume you were using this for `paigowpokerhi`? If so, you need to pass in the game string like:

```javascript
Hand.solve(['Ad', 'As', 'Ac', 'Th', '2d', '0r', '0r'], 'paigowpokerhi').descr
```

**Comment by acidicchip** [comment]
I was actually trying to use it to determine the highest possible (future) hand in Texas Hold'em by specifying 2 wildcards to represent what I have in my hand.

So ['Ad', 'As', 'Ac', 'Th', '2d', '0r', '0r'] would be 2 As in my hand, AT2 on the flop, and 2 variables (yet to come).

I figured it wasn't intended for that, but still thought it might had been a bug since one of them shows a full house as a winner, the other 4 of a kind, both contain 4+ of a kind.

**Comment by goldfire** [comment]
Okay, that seems to be a bug as there was no intended support for wildcards in the Texas Hold'em solver, but I can see how that use-case would be useful.

**Comment by ericblade** [comment]
There's certainly a functional difference between a wild card, and an unknown card.  There's no reason why someone *couldn't* play holdem with wilds (we've certainly done it in home games before :-) ) but a wild is a big difference to an unknown.  If there's already wild card support in the solver, I'd suggest allowing it, but that would also allow hands that wouldn't normally be possible (5 of a kind).  

... commenting as a programmer and a poker player, not as someone who's intimitely familiar with pokersolver, i have it watched just because i like it, not because i use it currently :-)

**Comment by acidicchip** [comment]
Yeah, there's definitely a difference between a wild and an unknown, but I was banking on the fact that a 5 of a kind doesn't exist in standard poker rules and expecting pokersolver to know that since I used the default hand settings.

**Comment by ericblade** [comment]
I guess what I'm saying, is that I'd recommend having both functioning wild-cards in all variants, and having a separate indicator for "Unknown", like maybe "??" or something.
